cpp/cgi_progs/Makefile

```diff
@@ -3,6 +3,8 @@ INC = $(LIB)/include
 include ../Makefile.inc
 PROGS = display_translate_stats full_text_cache_monitor full_text_lookup ini_validator translate_chainer translator translator_ajax zotero_cgi restart_zts message_dispatcher
 CGI_DIR = /var/www/cgi-bin
+APACHE_START=systemctl start apache2
+APACHE_STOP=systemctl stop apache2
```

Comment:
How about something like
`APACHE_SYSTEMCTL_ARGUMENT := $(shell if [ -f /etc/lsb_release ]; then echo "apache2"; elif [ -f /etc/centos-release ]; echo "httpd"; else echo "")`

This can be given as an argument to systemctl without further branching.

and abort if unset with something like:
```bash
ifndef MY_FLAG
.ABORT
endif
```